docs/zh_cn/conf.py

```diff
@@ -50,7 +50,6 @@
     'sphinx.ext.napoleon',
     'sphinx.ext.viewcode',
     'sphinx.ext.autosectionlabel',
-    'sphinx_markdown_tables',
```

Comment:
`sphinx_markdown_tables` is useless and conflicts with `myst`.
It also breaks the references in tables.